doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md

```diff
@@ -29,7 +29,7 @@ This guide describes how to upgrade your [Kaa server]({{root_url}}Glossary/#kaa-
 Follow these steps to run the upgrade script:
 
 1. Go to your Kaa host machine.
-2. Download the [upgradeKaa090to0100.sh](https://github.com/kaaproject/kaa/blob/v0.10.0/server/upgrade/data-migration-0.9.0-0.10.0/upgradeKaa090to0100.sh) and [data-migration_0.9.0-0.10.0.jar](http://repository.kaaproject.org/repository/releases/org/kaaproject/kaa/server/upgrade/data-migration/0.10.0/data-migration_0.9.0-0.10.0.jar) to upgrade the databases.
+2. Download the [upgradeKaa090to0100.sh]({{github_url}}server/upgrade/data-migration-0.9.0-0.10.0/upgradeKaa090to0100.sh) and [data-migration_0.9.0-0.10.0.jar](http://repository.kaaproject.org/repository/releases/org/kaaproject/kaa/server/upgrade/data-migration/0.10.0/data-migration_0.9.0-0.10.0.jar) to upgrade the databases.
```

Comment:
The first one should be {{github_url_raw}}.
The second link should be updated too.